index.html

```diff
@@ -47,11 +49,47 @@
           <div class="drop-bottom">
             <hr>
           </div>
-       
-       
-        
- </div>
+          </div>
+     
+          
+            <div class="popup2"  >
+              <h1 id="mob-pop"  >Multi post Stories</h1>
+              <h2 id="desk-pop" >Keeping track of hundreds  of components website</h2>
+             <div class="close-button-container2">
+               <button type="button"  class="close-button2" onclick="closePopup()"> &times;</button>
+             </div>
+             
+             <ul class="cardbtn">
+               <li><button type="button" class="middle-pop">html</button></li>
+               <li> <button type="button" class="middle-pop">bootstrap</button></li>
+               <li><button type="button" class="middle-pop">Ruby On Rails</button></li>
+               
+             </ul>
+             <div class="images-paragraph">
+              <img class="mobile-pop"  src="./assets/images/popup2.png" alt="">
+              <img class="desktop-pop" src="./assets/images/bigpop.png" alt="">
+             <div class="para-btn">
+               <div id="para">
+                <p >Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
+                  when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+                  It has survived not only five centuries,but also the leap into electronic typesetting, 
+                  remaining essent</p>
+               </div>
+              
+
+                <div class="popup-btn-container">
+                  <button type="button" class="popup-btn1">See Live</button>
+                  <button type="button" id="popBtn" class="popup-btn2">See Source</button>
+                </div>
+             </div>
+            </div>
```

Comment:
- [ ] Kindly render the popup window dynamically. it should pick the content from the project array at the specified index.  
- [ ] The image is the popup window is shrunk kindly fix the image display.
![Screen Shot 2022-04-15 at 11 28 46](https://user-images.githubusercontent.com/28307960/163554099-d5f53e6d-078e-439f-be49-c1b3b47beaab.png)